commit c512b7bd13c5838ded9741493267da05e1e49753 (refs/changes/31/3031/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2017-12-08T15:56:27+00:00 (1 year, 10 months ago)
    
    TRITON-23 booter should use node v4.x
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 23ed602..981d313 100644
--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,7 @@ CLEAN_FILES += ./node_modules build/pkg dhcpd-pkg-*.tar.bz2
 REPO_MODULES := src/node-pack
 JSSTYLE_FLAGS = -o indent=4,doxygen,unparenthesized-return=0
 
-NODE_PREBUILT_VERSION=v0.10.48
+NODE_PREBUILT_VERSION=v4.8.5
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
diff --git a/package.json b/package.json
index 3939add..b37563c 100644
--- a/package.json
+++ b/package.json
@@ -5,15 +5,15 @@
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-    "assert-plus": "0.1.2",
+    "assert-plus": "1.0.0",
     "bunyan": "1.8.8",
     "dashdash": "1.14.1",
     "findit": "2.0.0",
     "mkdirp": "0.5.0",
     "node-uuid": "1.2.0",
-    "restify": "2.8.4",
+    "restify": "4.3.0",
     "sdc-clients": "10.3.0",
-    "sdc-bunyan-serializers": "git+https://github.com/joyent/sdc-bunyan-serializers.git#5454ae4",
+    "sdc-bunyan-serializers": "git+https://github.com/joyent/sdc-bunyan-serializers.git#a3e7780",
     "sprintf": "0.1.0",
     "vasync": "1.6.2"
   },
@@ -26,8 +26,5 @@
   "scripts": {
     "test": "make test"
   },
-  "engines": {
-    "node": ">=0.10"
-  },
   "license": "MPL-2.0"
 }
